src/qt/rpcconsole.cpp

```diff
@@ -703,7 +703,7 @@ void RPCConsole::addWallet(WalletModel * const walletModel)
         // First wallet added, set to default so long as the window isn't presently visible (and potentially in use)
         ui->WalletSelector->setCurrentIndex(1);
     }
-    if (ui->WalletSelector->count() > 2) {
+    if (ui->WalletSelector->count() > 1) {
```

Comment:
Can we just directly count the number of wallets?

Reply:
This change doesn't make the code worse. I think this can be improved later by refactoring to take advantage of the new `WalletController`.